### PR TITLE
Fix duplicate tags bug and update fileAddress error message

### PR DESCRIPTION
**Pull request by anthony6401**

### CODE REVIEW

File: src/main/java/seedu/address/model/tag/FileAddress.java [hunk 1]
```diff
@@ -9,8 +9,7 @@
  */
 public class FileAddress {
 
-    // TODO change message to be more understandable
-    public static final String MESSAGE_CONSTRAINTS = "File addresses can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "File address can only take a file path!";
```
Comment (luo-git) [inline src/main/java/seedu/address/model/tag/FileAddress.java]: Maybe "valid" file path?

**Review by luo-git — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/model/tag/FileAddress.java (hunk 1 above)
Comment (anthony6401) [reply src/main/java/seedu/address/model/tag/FileAddress.java]: Changed!

**Review by HynRidge — APPROVED** [review]
LGTM

**Review by luo-git — APPROVED** [review]
LGTM

**Pull request merged**